### DISCUSSION TITLE
WW3 ICs outside of 2011-2018?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [NOAA-EMC/WW3#846] [Q&A] [answered]
I have been using WW3 as part of the development of the coupled UFS application, which is currently focused on the period from 2011-2018. I was wondering if initial conditions exist for a broader time range, ideally back to 2000. If so, where would I find them? If not does anyone have any suggestions for what else we could do (initialize with climatology, interpolate a different data source, ?). Thanks!

**Comment by JessicaMeixner-NOAA** [comment, marked as answer]
@benjamin-cash the typical strategy for generating initial conditions for the wave model is to simply run a wave model simulation with your given grid and set-up with your desired forcing for about 2weeks to a month.  This is how we obtained initial conditions for the 2011-2018 period you mentioned. There are details in the coupled prototype spreadsheet on how those were generated which aren't really relevant to the general WW3 github so I will not post the link here, email me offline if you need it but you should have access to that spreadsheet already.
